tools/src/main/java/org/apache/kafka/tools/reassign/MissingLogDirMoveState.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.tools.reassign;
+
+import java.util.Objects;
+
+/**
+ * A replica log directory move state where the source replica is missing.
```

Comment:
Is the description valid? Should we have to swap the description of `MissingLogDirMoveState` and `MissingReplicaMoveState`?

Reply:
I double checked and it seems that comments in java and scala code are the same.
Are you suggest that scala class comment is wrong?

Reply:
yes, it seems the class comments are wrong. I was going through the other PR, then noticed this.